packages/KStepper/KStepper.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div
+    :id="stepperId"
+    :style="stepperStyle"
+    class="k-stepper d-flex w-100"
+  >
+    <KStep
+      v-for="(step, idx) in steps"
+      :key="`k-step-${step.label.replace(' ', '-')}`"
+      :label="step.label"
+      :state="step.state"
+      :is-first="idx === 0"
+      :is-last="(steps.length - 1) === idx"
+      :step-size="stepSize"
+      :step-container-width="stepContainerWidth"
+      :divider-width="dividerWidth ? dividerWidth : undefined"
+      :margins="margins"
+    />
+  </div>
+</template>
+
+<script>
+import { uuid } from 'vue-uuid'
+import KStep from './KStep.vue'
+
+export default {
+  name: 'KStepper',
+  components: { KStep },
+  props: {
+    /**
+     * Array of steps to display
+     */
+    steps: {
+      type: Array,
+      required: true,
+      validator: (items) => !items.length || items.some(i => i.hasOwnProperty('label') && i.hasOwnProperty('state'))
+    },
+    maxWidth: {
+      type: String,
+      default: '100%'
+    },
+    labelWidth: {
+      type: String,
+      default: '170'
+    },
+    stepIconSize: {
+      type: String,
+      default: '24'
+    },
```

Comment:
Do we have a use-case for customizing the icon size? If not, I think we should remove this prop as it could lead to inconsistencies throughout Konnect

Reply:
Removed